mdadm.8.in

```diff
@@ -1830,6 +1830,17 @@ can be found it
 under
 .BR "SCRUBBING AND MISMATCHES" .
 
+.TP
+.B \-\-udev\-rules=
+Given the policies defined under
+.IR mdadm.conf
+it generates the udev rules to handle hot-plug bare devices
+
+See
+.BR mdadm.conf (5)
+for more details about policies.
```

Comment:
Please use keyword "POLICY" where possible. For less experienced reader it is easier to catch up if you are referring to the same context (and simple search by POLICY will work in mdadm.conf), for example:

```
Given the POLICY line/lines are defined under
.IR {CONFFILE}
```
{CONFFILE} can be used to set primary conf file location instead, if you want to. Not an issue for sure.

so far I remember `.IR` does not add something special but I could be wrong here. I'm not really oriented in man syntax.

Reply:
Please also mention that the "usage example" can be found in man mdadm.conf.

Reply:
added the {CONFFILE} and moved from `policies` to `POLICY`